People are confused enough about Cygwin already, we really don't want 
<Cocoa/Cocoa.h> and friends directly under /usr/include lest we be
mistaken for OS X. (FWIW, Debian does this as well.)

--- origsrc/gnustep-make-2.2.0/FilesystemLayouts/fhs-system	2008-01-16 20:24:05.000000000 -0600
+++ src/gnustep-make-2.2.0/FilesystemLayouts/fhs-system	2009-08-02 03:01:44.400720700 -0500
@@ -46,7 +46,7 @@ GNUSTEP_SYSTEM_WEB_APPS=/lib/GNUstep/Web
 GNUSTEP_SYSTEM_TOOLS=/bin
 GNUSTEP_SYSTEM_ADMIN_TOOLS=/sbin
 GNUSTEP_SYSTEM_LIBRARY=/lib/GNUstep
-GNUSTEP_SYSTEM_HEADERS=/include
+GNUSTEP_SYSTEM_HEADERS=/include/GNUstep
 GNUSTEP_SYSTEM_LIBRARIES=/lib
 GNUSTEP_SYSTEM_DOC=/share/GNUstep/Documentation
 GNUSTEP_SYSTEM_DOC_MAN=/share/man
@@ -58,7 +58,7 @@ GNUSTEP_NETWORK_WEB_APPS=/lib/GNUstep/We
 GNUSTEP_NETWORK_TOOLS=/local/bin
 GNUSTEP_NETWORK_ADMIN_TOOLS=/local/sbin
 GNUSTEP_NETWORK_LIBRARY=/local/lib/GNUstep
-GNUSTEP_NETWORK_HEADERS=/local/include
+GNUSTEP_NETWORK_HEADERS=/local/include/GNUstep
 GNUSTEP_NETWORK_LIBRARIES=/local/lib
 GNUSTEP_NETWORK_DOC=/local/share/GNUstep/Documentation
 GNUSTEP_NETWORK_DOC_MAN=/local/share/man
@@ -70,7 +70,7 @@ GNUSTEP_LOCAL_WEB_APPS=/local/lib/GNUste
 GNUSTEP_LOCAL_TOOLS=/local/bin
 GNUSTEP_LOCAL_ADMIN_TOOLS=/local/sbin
 GNUSTEP_LOCAL_LIBRARY=/local/lib/GNUstep
-GNUSTEP_LOCAL_HEADERS=/local/include
+GNUSTEP_LOCAL_HEADERS=/local/include/GNUstep
 GNUSTEP_LOCAL_LIBRARIES=/local/lib
 GNUSTEP_LOCAL_DOC=/local/share/GNUstep/Documentation
 GNUSTEP_LOCAL_DOC_MAN=/local/share/man
